src/main/sensors/gyro.c

```diff
@@ -474,23 +480,27 @@ FAST_CODE void gyroFiltering(timeUs_t currentTimeUs)
     }
 #endif
 
-    if (gyro.useDualGyroDebugging) {
+    if (gyro.useMultiGyroDebugging) {
         int debugIndex = 0;
 
-        for (int i = 0; i < GYRO_COUNT; i++) {
-            if (gyro.gyroEnabledBitmask & GYRO_MASK(i)) {
-                DEBUG_SET(DEBUG_DUAL_GYRO_RAW, 0 + debugIndex, gyro.gyroSensor[i].gyroDev.gyroADCRaw[X]);
-                DEBUG_SET(DEBUG_DUAL_GYRO_RAW, 1 + debugIndex, gyro.gyroSensor[i].gyroDev.gyroADCRaw[Y]);
-                DEBUG_SET(DEBUG_DUAL_GYRO_SCALED, 0 + debugIndex, lrintf(gyro.gyroSensor[i].gyroDev.gyroADC.x * gyro.gyroSensor[i].gyroDev.scale));
-                DEBUG_SET(DEBUG_DUAL_GYRO_SCALED, 1 + debugIndex, lrintf(gyro.gyroSensor[i].gyroDev.gyroADC.y * gyro.gyroSensor[i].gyroDev.scale));
+        // Iterate through the user-selected gyros for debugging
+        for (int selectionIndex = 0; selectionIndex < 4; selectionIndex++) {
```

Comment:
@coderabbitAI, i dont like the hardcoded `4`  what other option without adding complexity? apparently `GYRO_COUNT` is not longer usable here?